php/git.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+class Git {
+
+    /**
+     * Retrieve the commit ID of the current HEAD.
+     * @return string|null commit ID or null on failure
+     */
+    public static function getCurrentCommitID() {
+        $head = file_get_contents(".git/HEAD");
+
+        if (preg_match("/ref: (.*)/", $head, $matches)) {
```

Comment:
Not a bad thing to explicitly define `$matches` before the function

Reply:
done